src/main/steps/c100-rebuild/permission/content.ts

```diff
@@ -0,0 +1,94 @@
+import { YesOrNo } from '../../../app/case/definition';
+
+import { TranslationFn } from './../../../app/controller/GetController';
+import { FormContent } from './../../../app/form/Form';
+import { isFieldFilledIn } from './../../../app/form/validation';
+
+const en = {
+  title: 'Is there any reason why you would need permission from the court to make this application?',
+  courtPermissionLine1: 'In some cases, the court will need to give permission before the application can be made.',
+  courtPermissionLine2: 'You will not need permission if you are the:',
+  courtPermissionBullet1: 'parent',
+  courtPermissionBullet2: 'guardian',
+  courtPermissionBullet3: 'special guardian',
+  courtPermissionLine3: 'You may, or may not need permission from the court.',
+  courtPermissionLine4:
+    'For example, you may need permission if you are in the children’s lives, but not their parent or guardian.',
+  courtPermissionLine5:
+    'You will need the court’s permission if you are one of the children, and the application is about you.',
+  courtPermissionLine6:
+    'If you are not sure if you need permission, <a  target="_blank" href="https://www.gov.uk/government/publications/family-court-applications-that-involve-children-cb1">see Section C of the Form CB1 guidance on GOV.UK</a>',
+  summaryText: 'Contacts for help',
```

Comment:
@ShakiraKP Are you using this field 'summaryText' ?
Can you please if not you are not using it. Please avoid such unused lines in the code.